### PR TITLE
Using packaging.version.Version as LooseVersion

### PR DESCRIPTION
**Pull request by jlstevens**
An alternate approach to https://github.com/holoviz/holoviews/pull/5179: although the subclass approach in that PR would require fewer lines to be changed overall, this approach is more idiomatic in that no new classes are introduced.

Note that `packaging` is not expended to be a new dependency of HoloViews as HoloViews depends on Panel which in turn depends on Bokeh which pulls in `packaging`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "pyviz_comms >=0.7.4",
     "panel >=0.9.5",
     "colorcet",
+    'packaging',
```
Comment (philippjfr) [inline setup.py:19]: Consistency :)

```suggestion
    "packaging",
```

**Comment by philippjfr** [comment]
Thanks!

**Comment by jlstevens** [comment]
@philippjfr Now this PR has been rebased, the tests are going green. Ready for final review/merge.

**Comment by jlstevens** [comment]
The Python 3.8, macos-latest build failed but only because it hit the 60 minutes limit afaik.

**Pull request merged**